inductiva/_cli/cmd_logs/__init__.py

```diff
@@ -1,5 +1,6 @@
 """Register CLI commands for logs."""
 import os
+from inductiva import constants
 
 from inductiva._cli import loader, utils
```

Comment:
one empty line between builtin imports and no empty lines between custom imports:

```suggestion
import os

from inductiva._cli import loader, utils
from inductiva import constants
```